### ISSUE TITLE
Incremental learning

### ISSUE THREAD

**Issue by cassiehkx** [vi3k6i5/GuidedLDA#16]
It worked very well on small dataset. Can it be improved to enable incremental learning in case of a huge dataset?

**Comment by vi3k6i5** [comment]
@cassiehkx can you share link to some other form of LDA which has `incremental learning` or any other library?

Basically I want to understand how that works so I can think of an approach to do it.

Thanks :)

**Comment by cassiehkx** [comment]
I haven't got useful suggestions in this case to enable incremental learning. But the problem lies in your input as an nparray, which makes the program unable to expand to a large amount of data. Incremental learning is just one solution I could think of at the moment. Maybe we could change the input data into a sparse matrix? but in that case the matrix multiplication in the loglikelihood would be a problem. what would you recommend?

**Comment by vi3k6i5** [comment]
Fair point, switching to sparse matrix should be easier in comparison to incremental learning.

**Comment by tmerrittsmith** [comment]
In the Gensim implementation of LDA, you can set chunk size to learn incrementally I think?
